### ISSUE TITLE
How to use .env in react-native without using 3rd party libraries

### ISSUE THREAD

**Issue by AjayShivanagol** [facebook/react-native#31262]
I know there are good libraries out there to fix this(**react-native-dotenv, react-native-config**) all are 3rd party...so is there any official library that can solve this?

**Comment by ezranbayantemur** [comment]
Nope, community has **react-native-config** package and it works very well. 
So no need the official thing to do it.
